lib/cocoapods/sandbox/file_accessor.rb

```diff
@@ -172,6 +172,17 @@ def vendored_dynamic_frameworks
         end
       end
 
+      # @return [Array<Pathname>] The paths of the dynamic xcframework bundles
+      #         that come shipped with the Pod.
+      #
+      def vendored_static_xcframeworks
+        vendored_xcframeworks.select do |path|
```

Comment:
do these need to be `compact`?

Reply:
I guess not it seems it only selects the ones that are static frameworks.

Reply:
Confirmed. Thanks @amorde !